ft_safety_interrupt:
  interrupt_topic: "ft_interrupt"
  ft_topic: "filtered_force_torque"

  check_dirs: false
  direction_max_allowable: [100,100,100,0,0,0] # N, N, N, N-m, N-m, N-m
  
  check_mag: false
  magnitude_max_allowable: [0,0] # N, N-m

  # recovery direction will be converted to this frame  
  direction_output_frame: "bravo_base_link" 
  #direction_output_frame: "bravo_peg_link"

